packages/internal/generated-clients/Makefile

```diff
@@ -36,6 +36,7 @@ generate-mr-api-client:
 	rm -rf src/multi-rollup && \
     mkdir src/multi-rollup && \
 	docker run --rm -v $(shell pwd):/app openapitools/openapi-generator-cli:v7.0.1 generate \
+    --inline-schema-options REFACTOR_ALLOF_INLINE_SCHEMAS=true \
```

Comment:
When the cli was updated, it stopped combining `allOf` types.
https://immutable.atlassian.net/browse/GMS-1302

```
      "RefreshMetadataByTokenID": {
        "allOf": [
          {
            "$ref": "#/components/schemas/RefreshableNFTAttributes"
          },
          {
            "type": "object",
            "properties": {
              "token_id": {
                "type": "string",
                "description": "An `uint256` token id as string",
                "example": "1"
              }
            },
            "required": [
              "token_id"
            ]
          }
        ]
      },
```
Generated type for this would ignore `token_id`. 
This option on the cli allows that `allof` types to be created like before without need to downgrade it.